akka-actor/src/main/java/akka/dispatch/AbstractNodeQueue.java

```diff
@@ -14,7 +14,8 @@
  */
 @SuppressWarnings("serial")
 public abstract class AbstractNodeQueue<T> extends AtomicReference<AbstractNodeQueue.Node<T>> {
-    // Extends AtomicReference for the "head" slot (which is the one that is appended to) since Unsafe does not expose XCHG operation intrinsically
+    // Extends AtomicReference for the "head" slot (which is the one that is appended to) since
+    // Unsafe does not expose XCHG operation intrinsically before JDK 8
```

Comment:
That is true. But I’m not yet convinced that this is a strong enough reason to change.